Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/activities/EmulationActivity.java

```diff
@@ -417,6 +427,10 @@ protected void onResume()
   protected void onPause()
   {
     super.onPause();
+    if (VirtualReality.isActive())
+    {
+      System.exit(0);
+    }
```

Comment:
The entire app exits on `onPause`? I don't know VR devices very well, but from the perspective of regular Android development, this feels incredibly strange.

Reply:
This is a workaround for an ANR issue which randomly happens. I commented that into the code and modify this condition so it calls exit only if the headset is frozen.